### DISCUSSION TITLE
[BUG] No styling on block quote

### DISCUSSION THREAD

**Discussion by otajor** [mdx-editor/editor#449] [Q&A]
Just installed this today and it was very easy to get working and is a really smooth experience, awesome work. I'm just having a small issue with the quote styling that I don't understand.

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] I have read the documentation and cannot find an answer.

**Describe the bug**
Styling for quotes is not applied in my app, there is indentation but no italics, and no bar to the side like in the live demo. I can't see anything in the docs about adding extra styles here. I am just using the basic usage with the quotePlugin (see reproduction below). Is this expected?

On the live demo there is a section of styling that is applied to the blockquote that gives it the italics and left border:
<img width="329" alt="image" src="https://github.com/mdx-editor/editor/assets/13705650/9ec3264f-8890-40eb-9d06-dbf366f7f0f8">

These styles are not applied to my quotes. Is there something else I have to do to get them to show up?

**Reproduction**
[Reproduction here](https://codesandbox.io/p/sandbox/mdx-editor-base-forked-jq6q56?file=%2Fsrc%2FApp.tsx%3A20%2C9)

**To Reproduce**
Go to reproduction and check code and preview. Code shows markdown with quote block, preview shows markdown with quote block but missing styling.

**Expected behavior**
The quote block should include some styling like italic font-style and left-border to make clear it is a quote, like in the live demo (https://mdxeditor.dev/editor/demo)

**Screenshots**
Reproduction quote block:
<img width="315" alt="image" src="https://github.com/mdx-editor/editor/assets/13705650/336f365d-806b-434e-9084-0d8c8f240f1c">
Live demo quote block:
<img width="702" alt="image" src="https://github.com/mdx-editor/editor/assets/13705650/71ddcc8f-c818-41a2-a2b2-b698a621b3d1">

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser: Brave

**Comment by shedan9** [comment]
This problem still existed even after a year
